sql/sql_parse.cc

```diff
@@ -1437,7 +1437,7 @@ bool dispatch_command(THD *thd, const COM_DATA *com_data,
       else
       {
         mysql_reset_thd_for_next_command(thd);
-        my_error(ER_LOCK_DEADLOCK, MYF(0), "wsrep aborted transaction");
+        my_message(ER_LOCK_DEADLOCK, "WSREP cancelled the transaction", MYF(0));
```

Comment:
Yeah. Right. I thought over this and looks like it would invite more mess than solving it. Not good time for us to solve it. Ignore this.